packages/gatsby-source-npm-package-search/gatsby-node.js

```diff
@@ -0,0 +1,81 @@
+"use strict";
```

Comment:
This file shouldn't be checked in — copy the the .gitignore file from plop-templates/packages/gitignore.hbs

Reply:
The contents of the .gitignore in both places (plop-templates/packages/gitignore.hbs and packages/gatsby-source-npm-package-search/.gitignore) are the same. Is the handlebars file special in some way? Or would it be enough to just add `gatsby-node.js` to the .gitignore that's already there?

Reply:
you probably just need to run `git rm --cached gatsby-node.js` to remove it. If a file is checked in, even if it's now ignored by a .gitignore, it'll stay in Git until explicitly removed.